### PR TITLE
Clarify P1, P2, and P3 support levels

### PR DESCRIPTION
**Pull request by collinmcnulty**
We have quite specific expectations of what should be a ticket at each priority level. Especially for P1 tickets, our processes for handling these tickets only makes sense in the context of certain kinds of issues. Without this level of specificity, we do get tickets labeled higher than they should be because, understandably, everyone feels their own issue is high priority. This PR attempts to clarify on the existing process.

### CODE REVIEW

File: astro/astro-support.md [hunk 1]
```diff
@@ -45,12 +43,12 @@ Examples:
 
 Examples:
 
-- There is a bug in the Software UI.
-- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that affects your cluster's performance but isn't required to run task, such as changing the size of your cluster's database or adding a new VPC peering connection.
+- A DAG that has been deployed to an Astro deployment for the first time, or is on a local development environment, is unexpectedly not working.
```
[the remark this answers is not in the document]
Comment (collinmcnulty) [reply astro/astro-support.md:48]: good idea.

File: astro/astro-support.md [hunk 2]
```diff
@@ -25,17 +25,17 @@ The following are the best practices for submitting support requests in the Clou
 
 To help Astronomer support respond effectively to your support request, it's important that you correctly identify the severity of your issue. The following are the categories that Astronomer uses to determine the severity of your support request:
 
-**P1:** Critical impact, service is unusable in production.
+**P1:** Critical impact. A deployment is completely unavailable or a DAG that was previously working in production is no longer working.
 
-Examples:
+Because P1 tickets are handled with the highest levels of urgency, if Astronomer Support responds on a P1 ticket and subsequently does not hear back for 2 hours, the ticket priority will be changed to P2.
```
Comment (RNHTTR) [inline astro/astro-support.md:30]: > the ticket priority will be changed to P2.

I think we should say "will be automatically changed to P2."

File: astro/astro-support.md [hunk 3]
```diff
@@ -45,12 +45,12 @@ Examples:
...
 
-- There is a bug in the Software UI.
-- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that affects your cluster's performance but isn't required to run task, such as changing the size of your cluster's database or adding a new VPC peering connection.
+- A DAG that has been deployed to an Astro deployment for the first time, or is on a local development environment, is unexpectedly not working.
+- You need to [modify a Hybrid cluster setting](manage-hybrid-clusters.md) that affects your cluster's performance but isn't required to run tasks, such as changing the size of your cluster's database or adding a new VPC peering connection.
 - Astro CLI usage is impaired (for example, there are incompatibility errors between installed packages).
 - There is an Airflow issue that has a code-based solution.
 - You received a log alert on Astronomer.
-- You have lost use of a Public Preview feature that does not affect general services. 
+- You have lost use of a Public Preview feature that does not affect general services.
```
Comment (RNHTTR) [inline astro/astro-support.md:53]: nit: "does not affect generally available services"

File: software/support.md [hunk 4]
```diff
@@ -24,17 +24,15 @@ The following are the best practices for submitting support requests in the Astr
...
+**P1:** Critical impact, a deployment is completely unavailable or a DAG that was previously working in production is no longer working.
 
-Examples:
-
-- Your tasks are not running and restarting them didn't fix the issue.
-- Astronomer Software is experiencing an incident or downtime that is affecting your production data pipelines.
+Because P1 tickets are handled with the highest levels of urgency, if Astronomer Support responds on a P1 ticket and subsequently does not hear back for 2 hours, the ticket priority will be changed to P2.
 
-**P2:** High impact. Ability to use Astronomer Software is severely impaired but does not affect critical production pipelines.
+**P2:** High impact. Ability to use Astronomer Software is severely impaired but does not affect critical, previously working pipelines in production.
```
[the remark this answers is not in the document]
Comment (RNHTTR) [reply software/support.md:31]: also same comment... "automatically changed to a P2"

**Review by RNHTTR — CHANGES_REQUESTED** [review]

File: astro/astro-support.md (hunk 3 above)
Comment (collinmcnulty) [reply astro/astro-support.md:53]: I am actually not sure what this line is trying to say. I didn't edit it.

Comment (RNHTTR) [reply astro/astro-support.md:53]: I think we should edit it to link to [this doc](https://docs.astronomer.io/astro/feature-previews) which includes a blurb that preview features are not backed by an SLA.

File: astro/astro-support.md [hunk 5]
```diff
@@ -25,17 +25,17 @@ The following are the best practices for submitting support requests in the Clou
...
-- Your tasks are not running and restarting them didn't fix the issue.
-- Astronomer is experiencing an incident or downtime that is affecting your data pipelines in production.
+Additionally, if the immediate problem is solved but there are follow up investigations ongoing, those follow ups will be conducted in a separate ticket at a lower priority.
 
-**P2:** High impact. Ability to use Astro is severely impaired but does not affect critical pipelines in production.
+**P2:** High impact. Ability to use Astro is severely impaired but does not affect critical, previously working pipelines in production.
 
 Examples:
 
+- A newly deployed production DAG is not working, even though it was tested in another Astro Deployment (e.g. dev or test) beforehand.
```
Comment (jwitz) [inline astro/astro-support.md:38]:
```suggestion
- A newly deployed production DAG is not working, even though it ran successfully in a development or test environment.
```

File: astro/astro-support.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/astro-support.md:48]: Updated my suggestion, is this what you meant?

**Comment by collinmcnulty** [comment]
@jwitz With the changes you added, are you good to approve?

File: astro/astro-support.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (collinmcnulty) [reply astro/astro-support.md:48]: I'm removing the word "development" before Deployment. But otherwise yes, I like that.

**Comment by collinmcnulty** [comment]
@jwitz I think we're good now

**Review by jwitz — APPROVED** [review]
Me too! Let's merge it

**Pull request merged**